Sources/Common/DataModel/ImageData/index.js

```diff
@@ -564,10 +547,11 @@ export function extend(publicAPI, model, initialValues = {}) {
   vtkDataSet.extend(publicAPI, model, initialValues);
 
   if (!model.direction) {
-    model.direction = mat3.create();
+    model.direction = new Float64Array(9);
+    mat3.identity(model.direction);
   } else if (Array.isArray(model.direction)) {
     const dvals = model.direction.slice(0);
-    model.direction = mat3.create();
+    model.direction = new Float64Array(9);
```

Comment:
`model.direction = new Float64Array(model.direction)`
or, if you fear that direction.length > 9:
`model.direction = new Float64Array(model.direction.slice(0, 9))`